src/Controller/OrganizationController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Entity\CommissionableAsset;
+use App\Entity\Organization;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/organization")
+ */
+class OrganizationController extends AbstractController
+{
+    /**
+     * @Route("/home", name="organization_home", methods={"GET"})
+     */
+    public function home(): Response
+    {
+        $om = $this->getDoctrine()->getManagerForClass(Organization::class);
```

Comment:
EntityManager can be injected in the construct method or directly into the action method